IPCheck/dashAnalysis.sh

```diff
@@ -113,7 +113,8 @@ elif [ $KIND_OF = "Nodes" ]; then
 	echo ": DASH Analysis > " $ANALYSIS_RESULT_FILE
 	echo ": DASH Analysis > " $ANALYSIS_RESULT_FILE >> $LOG_FILE
 	java -jar $DASH_LIB $JSON_FILE -summary $ANALYSIS_RESULT_FILE >> $LOG_FILE
-	cp $SRC_DIR/package-lock.json $DEST_DIR
+	cp $SRC_DIR/package.json $DEST_DIR
+	cp $SRC_DIR/yarn.lock $DEST_DIR
```

Comment:
```suggestion
	cp $SRC_DIR/package-lock.json $DEST_DIR
```

I think this should be `package-lock.json` because a node project typically does not have a `yarn.lock` file.